index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>todo list</title>
+</head>
+<body>
+    <header>
+        <h1 id="header"></h1>
+    </header>
+    <main>
+        <section id="todos">
```

Comment:
Something that might look slightly cleaner here when appending children to one parent element would be to have your ```section``` element all on one line. 

Ex: ```<section id="todos"></section>```